lib/cc/config/json.rb

```diff
@@ -0,0 +1,27 @@
+module CC
+  module Config
+    class JSON
+      DEFAULT_PATH = ".codeclimate.json".freeze
+
+      def initialize(path = DEFAULT_PATH)
+        @path = path
+      end
+
+      def checks
```

Comment:
Any `Config::*` source object must have the same interface -- otherwise they can't be merged and used uniformly. So this will have to have all the same public methods as `Config::Default` and `Config::YAML`, but loaded from the JSON file.